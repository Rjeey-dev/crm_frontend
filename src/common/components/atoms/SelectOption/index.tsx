import React from 'react';

export interface IOption {
    value: string,
    text: string,
    selected?: boolean
}

function SelectOption(props: IOption) {
    return <option selected={props.selected} value={props.value}>{props.text}</option>;
}

export default SelectOption;