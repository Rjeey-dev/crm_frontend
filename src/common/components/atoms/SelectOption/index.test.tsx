import { shallow } from 'enzyme';
import React from 'react';

import SelectOption from './index';

it("Renders <SelectOption/> with correct properties", () => {
    const output = shallow(
        <SelectOption value='one' text='One'/>
    ).html();

    expect(output).toMatchSnapshot();
});
