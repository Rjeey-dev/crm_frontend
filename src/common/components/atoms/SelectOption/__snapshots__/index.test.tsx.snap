// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <SelectOption/> with correct properties 1`] = `"<option value=\\"one\\">One</option>"`;
